source/ContentScript/recorder.ts

```diff
@@ -170,6 +172,12 @@ class Recorder {
     });
   }
 
+  shouldRecord(element: HTMLElement): boolean {
+    if (!this.active) return this.active;
+    if (element.style.zIndex === '999999') return false;
```

Comment:
The target page might be using this value as well, wouldn't it be better to check the ID of the elements?

Reply:
yeah would be better. Will fix this.